### PR TITLE
Fix a11y issues and re-enable cypress ace checks again

### PR DESCRIPTION
**Pull request by jerolimov**
**Fixes**: 
https://issues.redhat.com/browse/ODC-5832

**Analysis / Root cause**: 
Our cypress includes some automatically accessibility checks via [cypress-axe](https://github.com/component-driven/cypress-axe). These tests found some issues on the add page, esp. the "Import from Devfile" and "Upload JAR file" flows.

**Solution Description**: 
* Enable auto. accessibility checks for the Devfile and upload jar file flows
* Add missing label to the "Java command" field (remove unnecessary form group)
* Add aria-hidden to catalog card image (as it just duplicates the label info below, also drop the displayName which is only "visible" in the edge case that the image could not be loaded. Now the visible information matches the a11y labels.

**Screen shots / Gifs for design review**: 
UI doesn't change.

AXE check - Import from Git without this PR:

import-from-git-before

AXE check - Import from Git with this PR:

import-from-git-after

AXE check - Import from Jarfile without this PR:

upload-jar-file-before

AXE check - Import from Jarfile with this PR:

upload-jar-file-after

**Unit test coverage report**: 
Unchanged

**Test setup:**
* Open developer perspective and run the "Import from Devfile" and "Upload JAR file" flows.
* Open the browser console / accessibility tools to inspect the rendered HTML code.
* I run the axe tests locally with the [axe - Web Accessibility Testing - Firefox extension](https://addons.mozilla.org/de/firefox/addon/axe-devtools/), (the chrome extensions doesn't work for me on localhost)

Cypress tests (see also screenshots above):
* `yarn test-cypress-devconsole` and run `create-from-devfile.feature`
* `yarn test-cypress-devconsole` and run `create-from-git.feature`
* `yarn test-cypress-devconsole` and run `upload-jar-file.feature`
* `yarn test-cypress-knative` and run `create-event-sources.feature`

cypress run Devfile feature:

cypress-devfile

Last test was marked as todo and doesn't work yet because one step is missing:

cypress-devfile-reason

cypress run Upload Jar file feature:

cypress-upload-jar-file

The failing tests are marked as todo and doesn't work yet because one step is missing:

cypress-upload-jar-file-reason

cypress Create from Git tests fails. But all errors are timeouts and are not related to this change.

create-from-git

cypress Create event sources also doesn't run successful. All tests fails with an timeout not related to this change.

create-event-sources

:confused: 

**Browser conformance**: 
- [x] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by jerolimov** [comment]
@makambalaji @gajanan-more  Can one of you verify if at least this cypress features works fine with this PR? Unlucky, I could not run them successfully with this PR or on master. I will investigate later why. Until then this would be really helpful. Thank you!

* yarn test-cypress-devconsole > create-from-devfile.feature
* yarn test-cypress-devconsole > create-from-git.feature
* yarn test-cypress-devconsole > upload-jar-file.feature
* yarn test-cypress-knative > create-event-sources.feature

File: frontend/packages/console-shared/src/components/formik-fields/item-selector-field/SelectorCard.tsx
Comment (spadgett) [inline frontend/packages/console-shared/src/components/formik-fields/item-selector-field/SelectorCard.tsx:28]: If the image is just decorative and the title is diplayed elsewhere on the card, this can just be `alt=""` (then you wouldn't need `aria-hidden`).

**Review by spadgett — COMMENTED** [review]
Thank you for fixing the a11y problems 👍

File: frontend/packages/console-shared/src/components/formik-fields/item-selector-field/SelectorCard.tsx
Comment (jerolimov) [reply frontend/packages/console-shared/src/components/formik-fields/item-selector-field/SelectorCard.tsx:28]: Thanks for the review. :)
The last time I added alt="" the reviewer said I should avoid this because it is also shown if the image is not loaded.. Both true. ;)

Comment (spadgett) [reply frontend/packages/console-shared/src/components/formik-fields/item-selector-field/SelectorCard.tsx:28]: I'm confused: what is shown? We're setting `alt` to an empty value, right? My understanding is that `alt=""` is preferred for decorative images that can be skipped by a screen reader.

Comment (spadgett) [reply frontend/packages/console-shared/src/components/formik-fields/item-selector-field/SelectorCard.tsx:28]: https://www.w3.org/WAI/tutorials/images/decorative/

> Decorative images don’t add information to the content of a page. For example, the information provided by the image might already be given using adjacent text, or the image might be included to make the website more visually attractive.
> 
> In these cases, a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers. Text values for these types of images would add audible clutter to screen reader output or could distract users if the topic is different from that in adjacent text. Leaving out the alt attribute is also not an option because when it is not provided, some screen readers will announce the file name of the image instead.

Comment (jerolimov) [reply frontend/packages/console-shared/src/components/formik-fields/item-selector-field/SelectorCard.tsx]: Thanks @spadgett. Rebased and just changed this to `alt=""`

**Review by spadgett — APPROVED** [review]
/lgtm
/hold since we're waiting for verification on the cypress changes

**Comment by makambalaji** [comment]
> @makambalaji @gajanan-more Can one of you verify if at least this cypress features works fine with this PR? Unlucky, I could not run them successfully with this PR or on master. I will investigate later why. Until then this would be really helpful. Thank you!
> 
> * yarn test-cypress-devconsole > create-from-devfile.feature
> * yarn test-cypress-devconsole > create-from-git.feature
> * yarn test-cypress-devconsole > upload-jar-file.feature
> * yarn test-cypress-knative > create-event-sources.feature

create-from-devfile.feature
image

create-from-git.feature
image

upload-jar-file.feature
image

No a11y issues are observed on localhost, After pr merge we can execute them on remote cluster.
Need to check, why its failing again [knative scripts]

**Comment by jerolimov** [comment]
:steam_locomotive: this PR is hop-on the merge train

Thanks Balaji, thanks Sam

**Pull request merged**
